sotodlib/qa/metrics.py

```diff
@@ -368,7 +388,7 @@ def _process(self, meta):
 
         obs_time = [meta.obs_info.timestamp] * len(tags)
         return {
-            "field": self._influx_field,
+            "field": f"{self._influx_field}{'_' + self._field_name.replace("_", "", 1) else ''}",
```

Comment:
Was this string formatting tested? I suspect it won't work. Maybe rather like this?
```suggestion
            "field": f"{self._influx_field}{'_' + self._field_name.replace('_', '', 1) if self._field_name else ''}",
```
Also, is there a reason to restrict the number of `'_'` to remove to one?

Oh now I see this is how it is above.

Reply:
Yeah looks like a typo.

I was removing any added potential extra underscores just for cleanliness of the field names in the influx db, but will just remove that for clarity.